packages/storybook-vue/src/stories/Textarea/TextareaStates.stories.mdx

```diff
@@ -0,0 +1,165 @@
+import { action } from "@storybook/addon-actions";
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { Textarea } from "@utrecht/component-library-vue";
+import { argTypes } from "./argTypes.ts";
+
+<Meta id="vue-textarea-states" title="Vue.js Component/Textarea/States" component={Textarea} argTypes={argTypes} />
+
+# Textarea States
+
+export const Template = (args, { argTypes }) => ({
+  props: Object.keys(argTypes),
+  components: { Textarea },
+  setup() {
+    return { args };
+  },
+  template: `
+  <FormField> 
+    <Textarea v-bind="args" @update:modelValue="action" :value="args.modelValue" />
+  <br/>
+  <br/>
+  <Paragraph v-if="args.modelValue"><strong> Textarea Binding:</strong> {{args.modelValue}}</Paragraph>
+  </FormField>
+  `,
+  methods: { action: action("modelValue") },
+});
+
+## Disabled
+
+<Canvas>
+  <Story
+    name="Disabled"
+    args={{
+      placeholder: "Default Textarea",
+      name: "default-textarea",
+      id: "default-textarea",
+      disabled: true,
+    }}
+    parameters={{
+      docs: {
+        source: {
+          code: `<Textarea placeholder="Disabled Textarea" name="disabled-textarea" id="disabled-textarea" :disabled="true" />`,
+        },
+      },
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+## Invalid
+
+<Canvas>
+  <Story
+    name="Invalid"
+    args={{
+      placeholder: "Invalid Textarea",
+      name: "invalid-textarea",
+      id: "invalid-textarea",
+      invalid: true,
+    }}
+    parameters={{
+      docs: {
+        source: {
+          code: `<Textarea placeholder="Invalid Textarea" name="invalid-textarea" id="invalid-textarea" :invalid="true"/>`,
+        },
+      },
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+## Read-only
+
+<Canvas>
+  <Story
+    name="Read-only"
+    args={{
+      placeholder: "Read-only Textarea",
+      name: "readonly-textarea",
+      id: "readonly-textarea",
+      readonly: true,
+    }}
+    parameters={{
+      docs: {
+        source: {
+          code: `<Textarea placeholder="Read-only Textarea" name="readonly-textarea" id="readonly-textarea" :readonly="true" />`,
+        },
+      },
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+## Required
+
+<Canvas>
+  <Story
+    name="Required"
+    args={{
+      placeholder: "Required Textarea",
+      name: "required-textarea",
+      id: "required-textarea",
+      required: true,
+    }}
+    parameters={{
+      docs: {
+        source: {
+          code: `<Textarea placeholder="Required Textarea" name="required-textarea" id="required-textarea" :required="true" />`,
+        },
+      },
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+## Focus
+
+<Canvas>
+  <Story
+    name="Focus"
+    args={{
+      placeholder: "Focus Textarea",
+      name: "focus-textarea",
+      id: "focus-textarea",
```

Comment:
For all these stories I think it would be more helpful to have an actual `value` (quick brown fox?) instead of a placeholder, this way we cannot test `...-focus-color`.

I suggest we add one separate story that shows `placeholder`.

Reply:
I think you didn't get the focus color because you didn't build the `component-library-vue`

![Screenshot 2022-08-01 at 13 34 26](https://user-images.githubusercontent.com/53454241/182139448-87db3df8-c641-40e9-91e2-bcad529145da.png)

From my prospect, I think each story has a placeholder with descriptive value, it's better than just text as value if it's only the focus issue.
WDYT?

Reply:
So, I will update all stories with a filled `modelValue`, the reason why is because we need to display the  `<Textarea />` font CSS option when the input's states changed!
Thanks, @Robbert  for the explanation

Reply:
Only the  Textarea `required` story I will leave the `modelValue` empty to demonstrate the state